src/main/java/dev/openfeature/javasdk/FlagEvaluationOptions.java

```diff
@@ -1,10 +1,13 @@
 package dev.openfeature.javasdk;
 
-import java.util.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
-import lombok.*;
+import lombok.Builder;
+import lombok.Singular;
 
-@Value
+@lombok.Value
```

Comment:
Not clear why this is being qualified rather than importing it as above? Is it because it's only used once?

Reply:
NM. It's now ambiguous. I get it.